opencti-platform/opencti-graphql/src/schema/general.js

```diff
@@ -41,6 +41,7 @@ export const buildRefRelationSearchKey = (type, field = ID_INTERNAL) => `${build
 export const CONNECTOR_INTERNAL_ENRICHMENT = 'INTERNAL_ENRICHMENT'; // Entity types to support (Report, Hash, ...) -> enrich-
 export const CONNECTOR_INTERNAL_IMPORT_FILE = 'INTERNAL_IMPORT_FILE'; // Files mime types to support (application/json, ...) -> import-
 export const CONNECTOR_INTERNAL_EXPORT_FILE = 'INTERNAL_EXPORT_FILE'; // Files mime types to generate (application/pdf, ...) -> export-
+export const CONNECTOR_INTERNAL_NOTIFICATION = 'INTERNAL_NOTIFICATION'; // Files mime types to generate (application/pdf, ...) -> export-
```

Comment:
The comment doesn't seem right : `// Files mime types to generate (application/pdf, ...) -> export-`